pycore/torchparse.py

```diff
@@ -0,0 +1,72 @@
+import sys
+
+from torchinfo import summary
+
+import pycore.tikzeng as pnn
+
+
+class TorchArchParser:
+
+    text_mapping = {
+        "Linear": "\\mathrm{{FC}}",
+        "ReLU": "\\varphi_\\mathrm{{ReLU}}"
+    }
+
+    def __init__(self, torch_module, input_size):
+
```

Comment:
You could add an `assert` statement to make sure the class parameters are not `None` and the correct type.

Reply:
See 58a67b64ad603d8422cd4d174d2185c9c7357bd1